app/models/choice.rb

```diff
@@ -2,6 +2,8 @@ class Choice < ActiveRecord::Base
   belongs_to(:answer, :inverse_of => :choices, :touch => true)
   belongs_to(:option, :inverse_of => :choices)
 
+  validates :answer, :option, presence: true
```

Comment:
I noticed we aren't using shoulda-matchers to check this kind of thing on specs. And i didn't see a model spec for `Choice`... Should i just leave it like this?